app/assets/javascripts/utils/language_names.js

```diff
@@ -573,10 +697,18 @@ const LanguageNames = {
       name: 'Sinhala',
       nativeName: 'සිංහල',
     },
+//    'simple_form.en': { // does it have to be there?
```

Comment:
We have a `simple_form.en` locale file, but it's not for Simple English. I think it's used by a dependency called 'simple_form', and was generated automatically when that was installed. But it doesn't show up as an available locale and we don't need it in this file.

Reply:
removed in the last checkout